types/react-native-auth0/index.d.ts

```diff
@@ -324,7 +328,7 @@ export class Auth0ContextInterface {
     error: BaseError | null;
     isLoading: boolean;
     authorize(parameters?: AuthorizeParams, options?: AuthorizeOptions): Promise<void>;
-    clearSession(): Promise<void>;
+    clearSession(parameters?: ClearSessionParams, options?: ClearSessionOptions): Promise<void>;
```

Comment:
Is this method really present on the Auth0ContextInterface? It is not described in https://auth0.github.io/auth0-react/interfaces/Auth0ContextInterface.html

Reply:
@marknelissen This seems to refer to `auth0-react` which is a different SDK. We are updating the types for `auth0-react-native` here. The doc to that is - https://auth0.github.io/react-native-auth0/global.html#useAuth0